Cargo.toml

```diff
@@ -42,6 +42,10 @@ ignore = "0.2"
 serde = "1.0"
 serde_yaml = "0.7"
 
+[dependencies.sass-rs]
```

Comment:
Looks like you aren't fully rebased against master.  Want me to take care of that when the review is done?

Reply:
I need to learn git more, never done PR before :-/ How do I rebase my fork/master to upstream/master and then sync it to fork/dev to verify everything's fine, and then merge to fork/master in order to make a PR?

Reply:
And it seems my git is messing around line feeds, everytime I'm running rustfmt all the files looked liked modified but nothing has changed